docs/API.md

```diff
@@ -47,9 +47,10 @@
   * `extreme` (boolean): Enables extreme mode, yields an additional 60% performance
     (from 250ms down to 100ms per 10000 ops). There are trade-off's should be
     understood before usage. See [Extreme mode explained](extreme.md). Default: `false`.
-  * `level` (string): one of `'fatal'`, `'error'`, `'warn'`, `'info`', `'debug'`,
-    `'trace'`; also `'silent'` is supported to disable logging. Any other value
+  * `level` (string|object): one of `'fatal'`, `'error'`, `'warn'`, `'info`', `'debug'`,
+    `'trace'`; also `'silent'` is supported to disable logging. Any other string value
     defines a custom level and requires supplying a level value via `levelVal`.
+    Alternatively an object can be provided to specify different levels for (future) child loggers
```

Comment:
Missing a period at the end of the sentence. I'd also like a little more detail. As it is, I really don't know what it even means.